docs/slack.rst

```diff
@@ -49,11 +49,22 @@ Here are a few ways to preserve what you've learned for yourself and others:
 .. admonition:: Why we don't pay for Slack
    :class: note
 
-   The Write the Docs Slack community is free-to-access and we aim to keep it that way.
-   A Slack subscription is prohibitively expensive—over $10,000 per month—for a community like ours.
+   The Write the Docs Slack community is free to access and we aim to keep it that way.
+   A Slack subscription is prohibitively expensive—over $10,000 per month—for a community as large as ours.
    Even if we had such resources, upgrading Slack is not a priority
    and many people feel a freedom in knowing that their words will disappear shortly.
 
+Why not use a tool like Discord?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In related discussions, people sometimes ask about moving the community to a different tool, like Discord, so we can access more chat history. 
+We haven't considered that for a few reasons:
+
+- Because Write the Docs is a work-related community, a lot of people access it from their work computers, and often use it alongside their professional Slack instances.
+  Conversely, many organizations restrict social media sites like Discord on their devices.
+- Our community has a *lot* of people in it - we're coming up on 20,000 registered members.
+  It's hard to move an active community that large to a different platform, especially people who aren't around very often.
```

Comment:
Three things here:

- Like above, I'm suggesting leading with the issue, and then filling in the details.
- Hard is subjective and I don't want folks to try to debate us on this. Suggesting a bit of a reframing here.

```suggestion
- Migrating platforms is not a priority for the capacity of our small moderation team.
  Our community has a *lot* of people in it - we're coming up on 20,000 registered members.
  Moving an active community that large to a different platform requires a lot of attention that we believe is better spent on other tasks to serve the community.
```